pivotaltracker.properties

```diff
@@ -1,11 +1,10 @@
-
 implicitWait=30
 explicitWait=40
 waitSleepTime=500
 
```

Comment:
please add a space before and after to equal sign

Reply:
done